src/N3Store.js

```diff
@@ -357,20 +355,26 @@ export default class N3Store {
   // ### `getQuads` returns an array of quads matching a pattern.
   // Setting any field to `undefined` or `null` indicates a wildcard.
   getQuads(subject, predicate, object, graph) {
+    return [...this._yieldQuads(subject, predicate, object, graph)];
+  }
+
+  // ### `_yieldQuads` returns an generator of quads matching a pattern.
+  // Setting any field to `undefined` or `null` indicates a wildcard.
+  *_yieldQuads(subject, predicate, object, graph) {
```

Comment:
Maybe 
```suggestion
  *readQuads(subject, predicate, object, graph) {
```
and make this public.

Reply:
in a4cf8e7610bf6e130c30c9542035d078cddcb4c6